lib/membrane/aac/parser/helper.ex

```diff
@@ -155,6 +155,6 @@ defmodule Membrane.AAC.Parser.Helper do
         sample_rate: AAC.sampling_frequency_id_to_sample_rate(sr_index),
         channels: AAC.channel_config_id_to_channels(channel_configuration),
         encapsulation: :none,
-        samples_per_frame: if(frame_length_flag == 1, do: 1024, else: 960)
+        samples_per_frame: if(frame_length_flag == 0, do: 1024, else: 960)
```

Comment:
Maybe, to avoid similar problems you could also match the size of `rest` in the function clause?

Reply:
Unfortunately, the size of `rest` can vary. For starters, when second flag in GASpecificConfig is set, an additional value is inserted into before the next flag and everything gets shifted by 14 bits. Moreover, GASpecificConfig can be extended with additional information that we do not care about (for now). Actually, we do not care about anything after frameLengthFlag. There is a lot of possible combinations and implementing all of them seems a bit unnecessary at this point.

Reply:
As discussed -> forget about it